jaggery-modules/store/scripts/server.js

```diff
@@ -103,25 +103,42 @@ var server = {};
      * @param session
      */
     server.tenant = function (request, session) {
-        var obj, domain, user,
+        var obj, domain, user, urlDomain,
             carbon = require('carbon');
-        /*matcher = new URIMatcher(request.getRequestURI());
-         if (matcher.match('/{context}/' + opts.tenantPrefix + '/{domain}') ||
-         matcher.match('/{context}/' + opts.tenantPrefix + '/{domain}/{+any}')) {
-         domain = matcher.elements().domain; */
+        matcher = new URIMatcher(request.getRequestURI());
```

Comment:
Can you make this line:

var matcher = new URIMatcher(request.getRequestURI());

*Note:* This is an issue which was not introduced by this PR but rather a long standing problem